### DISCUSSION TITLE
2024-08-08 목요일

### DISCUSSION THREAD

**Discussion by Captain-Kim** [nbc-final-a10/TripBuddy#238] [기록보드]
2024-08-08 목요일 기록보드입니다.

## 여행가고십조(A10) 일일 공통 일과표

| 시간 | 일과 |  기록 |
| :-: | :-: | :-: |
| 09:00 ~ 10:00 | 아침 회의 | |
| 10:00 ~ 11:00 | 기술면접 준비 | |
| 11:00 ~ 13:00 | 집중 개발 time | |
| 13:00 ~ 14:00 | 점심 식사 | |
| 14:00 ~ 18:00 | 집중 개발 time | |
| 18:00 ~ 19:00 | 저녁 식사 | |
| 19:00 ~ 20:00 | 저녁 회의 | |
| 20:00 ~ 21:00 | 5분 기록보드 작성 / 개인 회고 | |

```
## 개인 기록보드 서식 (김병준)

### 오늘 한 일

- 내용

### 트러블 슈팅

- 내용

### 내일 계획

- 내용
```

```
// 이미지 테이블
<table>
  <tr>
    <th>이미지 1번 제목</th>
    <th>이미지 2번 제목</th>
  </tr>
  <tr>
    <td><img src="이미지 주소" width="300" height="300"/></td>
    <td><img src="이미지 주소" width="300" height="300"/></td>
  </tr>
</table>
```

**Comment by Captain-Kim** [comment]
## 개인 기록보드 서식 (김병준)

### 오늘 한 일

- 리치텍스트 에디터 구현 시작
- 온도지수 계산 로직 구현 시작

### 트러블 슈팅

- 학습 목적에서는 좋지만 우리 사이트에서 이미지 처리가 꼭 필요한가?
- 글 수정 로직이 퍼널 패턴에서 너무 복잡하다

### 내일 계획

- 리치텍스트 에디터 구현 ~ing
- 온도지수 계산 로직 구현 ~ing

**Comment by Captain-Kim** [comment]
**유저 테스트 슬렉 메시지**

![CleanShot 2024-08-08 at 11 10 26@2x](https://github.com/user-attachments/assets/b223068f-8a22-40d3-bc8f-c8338de3acb2)

**Comment by eunohhh** [comment]
## 개인 기록보드 서식 (오은)

### 오늘 한 일

- [x] : 스토리 글쓰기 시 옵션 조금 상세하게 - 완료
- [x] : 튜토리얼 데스크탑에서 헤더 숨기기 - 완료
- [x] : pwa 연구 - 완료
- [x] : 스토리 삭제 - 완료
- [x] : 스토리 좋아요 - 완료
- [ ] : 스토리 좋아요 옵티미스틱 - 예정
- [x] : 모바일에서 스토리 인풋문제 - 완료
- [ ] : 네이버 검수요청 - 예정
- [ ] : 미들웨어 설정 - 예정
- [ ] : 여정 찜하기 - 예정
- [ ] : 데스크탑 홈에서 마우스 스크롤 동작 수정 - 예정

### 트러블 슈팅

- pwa 설치 유도 문구를 ios 에서도 보여지게 하기 위해 navigator.share 를 사용하였으나 되지 않는 것을 최종 확인

### 내일 계획

- 안한 것들 마저 하기~

**Comment by slsl2** [comment]
## 개인 기록보드 서식 (김민지)

### 오늘 한 일

- notifications 페이지 UI 작업
- trips 페이지에서 Alarm 아이콘 누르면 notifications 페이지로 이동
- 탭메뉴 글씨 추가, 컬러 변경

### 트러블 슈팅

- 안 읽은 메시지 숫자 표시하는 방법에 대한 고민. => contract 테이블에 '마지막으로 읽은 메시지 아이디' 컬럼 추가하는 방법으로 결정

### 내일 계획

- 안 읽은 메시지 표시 기능 구현 시작
- 채팅 페이지 리팩토링...
